Presentation.Web/Models/API/V2/Internal/Request/Organizations/OrganizationGridConfigurationRequestDTO.cs

```diff
@@ -1,8 +1,9 @@
-﻿using Core.DomainModel;
-using System;
+﻿using System;
 using System.Collections.Generic;
+using Core.DomainModel;
+using Presentation.Web.Models.API.V1;
```

Comment:
don't reference V1 from V2, we want to separate those.

Create a new V2 type for the KendoColumnConfigurationDTO